src/main/resources/static/css/style.css

```
@@ -32,9 +40,21 @@ nav {
 .logo {
     font-weight: 800;
     font-size: 1.6rem;
-    color: var(--text-color);
     text-transform: uppercase;
     letter-spacing: 2px;
+
+    background: linear-gradient(90deg, #ffffff, #d4af37, #ffffff);
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)